code/_onclick/click.dm

```diff
@@ -274,20 +274,20 @@
 	return
 
 /*
-	Middle click
+	Shift Middle click

```

Comment:
Shift click is not used for swapping hands in your implementation, from what I can tell.

Reply:
Fixed the note to correctly state what the function is now.